tests/traders/test_trader.py

```diff
@@ -483,21 +484,23 @@ class TestTrader:
             PORTFOLIO_TOTAL: 1000
         }
 
-        with patch.object(PricesManager, 'get_mark_price', new=AsyncMock(return_value=1)):
-            orders = await trader_inst.sell_all()
+        if not os.getenv('CYTHON_TEST_IGNORE'):
```

Comment:
sad to have to do this but if no choice then I guess it's best this way

Reply:
Yes I tested may ways to avoid doing this...